apps/main/models.py

```diff
@@ -92,7 +94,11 @@ def __str__(self):
     def image_exists(self):
         if not self.image_url:
             return False
-        r = requests.head(self.image_url)
+        if not (str(self.image_url).endswith('.jpeg')
+                or str(self.image_url).endswith('.jpg')
+                or str(self.image_url).endswith('.png')):
+            return False
```

Comment:
Not Pythonic way. Proposal:

```
extensions = ['.jpeg', '.jpg', '.png']
return any([self.image_url.endswith(ext) for ext in extensions])
```

Reply:
@leocoan please read about built-in `any`. It returns bool value, so you do not need to use `if` here.